CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 2.16.1 / 2024-12-17
+
+* [IMPROVEMENT] Refactoring for React Native Session Replay support. 
+  See [#2448](https://github.com/DataDog/dd-sdk-android/pull/2448)
```

Comment:
minor: I don't think having `See ...` on a new line is rendered as expected for the such bullet list. It seems it is shown with empty indent, not with the one list item has.

Reply:
👍  We did this for 2.16.0, but I can go back to the original format